components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/reporting-overview.component.ts

```diff
@@ -83,41 +91,44 @@ export class ReportingOverviewComponent implements OnInit, OnDestroy {
 
   onSelectedTab(event) {
     this.selectedButtonTab = event.target.value;
-    this.getData(this.reportQuery.filters.getValue());
+    this.getData(this.reportQuery.getReportQuery());
   }
 
   onTrendIntervalChange(event) {
     const interval = event.target.value;
-    const {intervals, endDate} = this.reportQuery;
-    const startDate = intervals[interval][1](endDate);
-    this.reportQuery.interval = interval;
-    this.reportQuery.setDateRange(startDate, endDate);
+    const queryParams = {...this.route.snapshot.queryParams};
+    if (interval === '0' ) {
+      delete queryParams['date_interval'];
+    } else {
+      queryParams['date_interval'] = interval;
+    }
+
+    this.router.navigate([], {queryParams});
```

Comment:
To update the date range interval, set 'date_interval' equal to the updated index in the URL.